inputFiles/singlePhaseFlow/thermalCompressible_2d_base.xml

```diff
@@ -51,12 +51,22 @@
 
 	<!-- SPHINX_SolidInternalEnergy -->
     <SolidInternalEnergy
-      name="rockInternalEnergy"
-      volumetricHeatCapacity="4.56e6"
+      name="rockInternalEnergy_linear"
+      referenceVolumetricHeatCapacity="4.56e6"
+      dVolumetricHeatCapacity_dTemperature="0"
       referenceTemperature="0"
       referenceInternalEnergy="0"/>
 	<!-- SPHINX_SolidInternalEnergyEnd -->
 
+    <!-- SPHINX_SolidInternalEnergy_nonLinear -->
+    <SolidInternalEnergy
+      name="rockInternalEnergy_nonLinear"
```

Comment:
suggest to use the name as `rockInternalEnergy_linear`

Reply:
see comment above